ta/pkcs11/src/pkcs11_token.c

```diff
@@ -80,3 +81,96 @@ void pkcs11_deinit(void)
 	for (id = 0; id < TOKEN_COUNT; id++)
 		close_persistent_db(get_token(id));
 }
+
+uint32_t entry_ck_slot_list(uint32_t ptypes, TEE_Param *params)
+{
+	const uint32_t exp_pt = TEE_PARAM_TYPES(TEE_PARAM_TYPE_MEMREF_INOUT,
+						TEE_PARAM_TYPE_NONE,
+						TEE_PARAM_TYPE_MEMREF_OUTPUT,
+						TEE_PARAM_TYPE_NONE);
+	TEE_Param *out = &params[2];
```

Comment:
same here:  `out = param + 2`?    and other occurences...

Reply:
This is OK.